stan/math/prim/prob/exp_mod_normal_cdf.hpp

```diff
@@ -48,59 +48,55 @@ return_type_t<T_y, T_loc, T_scale, T_inv_scale> exp_mod_normal_cdf(
   scalar_seq_view<T_inv_scale> lambda_vec(lambda);
   size_t N = max_size(y, mu, sigma, lambda);
 
-  for (size_t n = 0; n < N; n++) {
-    if (is_inf(y_vec[n])) {
-      if (y_vec[n] < 0.0) {
-        return ops_partials.build(0.0);
-      }
+  for (size_t n = 0, size_y = size(y); n < size_y; n++) {
+    if (is_inf(y_vec[n]) && y_vec[n] < 0.0) {
+      return ops_partials.build(0.0);
     }
+  }
 
+  for (size_t n = 0; n < N; n++) {
     const T_partials_return y_dbl = value_of(y_vec[n]);
     const T_partials_return mu_dbl = value_of(mu_vec[n]);
     const T_partials_return sigma_dbl = value_of(sigma_vec[n]);
     const T_partials_return lambda_dbl = value_of(lambda_vec[n]);
     const T_partials_return u = lambda_dbl * (y_dbl - mu_dbl);
     const T_partials_return v = lambda_dbl * sigma_dbl;
-    const T_partials_return v_sq = v * v;
-    const T_partials_return v_over_sqrt_two = v / SQRT_TWO;
+    const T_partials_return v_over_sqrt_two = v * INV_SQRT_TWO;
+    const T_partials_return inv_sigma = 1 / sigma_dbl;
```

Comment:
[optional] There's a specialized function, `stan::math::inv`, that has analytic gradients. It's not going to be a large savings, but thought I'd mention it anyway.

Reply:
I'm switching to it.